### ISSUE TITLE
Document points of cooperation

### ISSUE THREAD

**Issue by jrochel** [ocsigen/lwt#796]
I'm having trouble extracting from the documentation and/or the code at which points during the execution of a thread the scheduler regains control to (potentially) pass it on to another thread.

The only explicit mentions of control being yielded are `Lwt_main.yield` and `Lwt.pause`. I'm certain that there are many more implicit points of cooperation (such as every `bind`?). Could anyone please shed some light on this?

**Comment by aantron** [comment]
Have you read [Execution model](https://ocsigen.org/lwt/5.2.0/api/Lwt#3_Executionmodel)? Can you rephrase your question in its terms?

**Comment by jrochel** [comment]
Indeed it seems that I hadn't (weirdly enough after having been using lwt for years). Thanks for pointing it out. I suppose that the documentation could be a bit more explicit about which exact functions do suspend execution, but I'm not complaining. Thanks again!

**Comment by yuri-kane-carter** [comment]
Bonjour Mr Rochel

Désole aucun rapport, mais j'ai retrouvé votre porte feuille ce matin au rayon Fruits et Légumes du Biocoop Belleville ce mardi 18. Ils avaient du oublié de le ramener à l'épicerie avec les autres objets perdus.
Je me tiens a votre disposition jusqu'a 15h ou je finis, je le laisserais à l'accueuil de l'épicerie.
06 38 35 60 86

**Comment by jrochel** [comment]
C'est trop cool. Merci encore.

**Comment by dannywillems** [comment]
@jrochel On se demande si c'est un fake :joy:. Ca semble trop beau pour être vrai.
